validator/client/log.go

```diff
@@ -45,16 +45,23 @@ func (v *validator) LogNextDutyTimeLeft(slot uint64) error {
 	}
 
 	var nextDutySlot uint64
-	var role string
+	attestingCount := 0
+	proposingCount := 0
 	for _, duty := range v.duties.CurrentEpochDuties {
-		if duty.AttesterSlot > slot && (nextDutySlot > duty.AttesterSlot || nextDutySlot == 0) {
+		if duty.AttesterSlot == nextDutySlot {
+			attestingCount += 1
```

Comment:
Just do `len(duty.Committee)` for attesting count

Reply:
Is `duty.Committee` contains all the validator indices in the committee? and `len` of it means number of validators in the committee?

The `attestingCount` here is the number of the user-provided pubkeys involved for attesting in next duty slot.